src/OrderFormAcceptanceTests.Steps/Utils/EnvironmentVariables.cs

```diff
@@ -67,6 +68,30 @@ private static string JsonConfigValues(string section, string defaultValue)
             return string.IsNullOrEmpty(result) ? defaultValue : result;
         }
 
+        internal static User User(UserType userType)
+        {
+            string userTypeString = string.Empty;
+            string defaultUserName = string.Empty;
+
+            switch (userType)
+            {
+                case UserType.Authority:
```

Comment:
is authority right in this context?

Reply:
It was either authority or NonBuyer, I thought we would be able to use authority for future tests